app/controllers/user_profile/about_me_controller.rb

```diff
@@ -0,0 +1,46 @@
+# -*- encoding : utf-8 -*-
+class UserProfile::AboutMeController < ApplicationController
+  before_filter :set_title
+  before_filter :check_user_logged_in
+
+  def edit ; end
+
+  def update
+    if @user.banned?
+      flash[:error] = _('Banned users cannot edit their profile')
+      redirect_to edit_profile_about_me_path
+      return
+    end
+
+    @user.about_me = params[:user][:about_me]
+
+    if @user.save
+      if @user.profile_photo
+        flash[:notice] = _("You have now changed the text about you on your profile.")
+        redirect_to user_url(@user)
+      else
+        flash[:notice] = _("<p>Thanks for changing the text about you on your " \
+                           "profile.</p><p><strong>Next...</strong> You can " \
+                           "upload a profile photograph too.</p>")
+        redirect_to set_profile_photo_url
+      end
+    else
+      render :edit
+    end
+  end
+
+  private
+
+  def check_user_logged_in
+    if authenticated_user.nil?
+      flash[:error] = _("You need to be logged in to change the text about you on your profile.")
+      redirect_to frontpage_url
+      return
+    end
+  end
+
+  def set_title
+    @title = _('Change the text about you on your profile at {{site_name}}',
```

Comment:
This was just what the [previous view](https://github.com/mysociety/alaveteli/blob/develop/app/views/user/set_profile_about_me.html.erb#L1-L2) was using, so was avoiding a `msgstr` change. Going to stick with this unless you think its really worth changing?